### PR TITLE
ID-1: Edit readme. Fix webhook create args

### PR DESCRIPTION
**Pull request by dmitrijivanchenko**
### [Documents conversion](https://uploadcare-evrone.atlassian.net/browse/UC-1)

- Add docs for:
  - copy File
  - list/update/delete Webhook
  - store Group
- Fix webhooks create method args

## Checklist

- [ ] Tests (if applicable)
- [ ] Documentation (if applicable)
- [ ] Changelog stub (or use [conventional commit messages](https://www.conventionalcommits.org/))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,6 +179,8 @@ File entity contains its metadata.
   "https://api.uploadcare.com/files/FILE_ID_IN_YOUR_PROJECT/",
  "uuid"=>"8f64f313-e6b1-4731-96c0-6751f1e7a50a"}
 
+@file.store # copies file, returns a new (copied) file metadata
```
Comment (rsedykh) [inline README.md:182]: `@file.copy`?

**Review by rsedykh — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dmitrijivanchenko) [reply README.md:182]: :+1:

**Review by rsedykh — APPROVED** [review]

**Review by dmitry-mukhin — APPROVED** [review]

File: lib/uploadcare/client/webhook_client.rb [hunk 2]
```diff
@@ -9,8 +9,12 @@ module Client
     class WebhookClient < RestClient
       # Create webhook
       # @see https://uploadcare.com/docs/api_reference/rest/webhooks/#subscribe
-      def create(target_url, event: 'file.uploaded', is_active: true)
-        body = { 'target_url': target_url, 'event': event, 'is_active': is_active }.to_json
+      def create(options = {})
+        body = {
+          'target_url': options[:target_url],
+          'event': options[:event] || 'file.uploaded',
+          'is_active': options[:is_active] || true
```
Comment (evgeniy-r) [inline lib/uploadcare/client/webhook_client.rb:16]: It seems that we get `"is_active": true` if we send `is_active: false`. Is this OK?

Comment (evgeniy-r) [reply lib/uploadcare/client/webhook_client.rb:16]: Maybe use
```ruby
'is_active': options[:is_active].nil? ? true : options[:is_active]
```

Comment (evgeniy-r) [reply lib/uploadcare/client/webhook_client.rb:16]: And possibly we should add a test for this case.

**Review by evgeniy-r — APPROVED** [review]

**Pull request merged**